SU-SWS/stanford_paragraph_types
# Hsdo 923 Hsdo 946 teaser colors

# READY FOR REVIEW

# Summary
- Added Color and texture Options. More textures might come or the textures might change after HSDO review. But that would be a simple change.

# Needed By (Date)
- End of sprint

# Urgency
- Low

# Steps to Test

1. Checkout branch & enable stanford_story_page/revert stanford_paragraph_types
2. Create story page and create a paragraph item of type "Teaser Cards"
3. Open the fieldset "Add Some Color" and choose a color and a texture (both are needed at this time - if only one is chosen, neither will be applied.)
4. verify the title below the image has a color background with a texture near the bottom before the rest of the fields.
